src/vendor/github.com/xelabs/go-mysqlstack/sqlparser/sql.y

```diff
@@ -1365,72 +1365,71 @@ radon_statement:
     $$ = &Radon{ Action: ReshardStr, Table: $3, NewName: $5}
   }
 
-show_statement_type:
```

Comment:
`show_statement_type ` is an abstract for str "show str force_eof",  str can be ID,reserved_keyword and non_reserved_keyword. I think this is more clear for code.

Reply:
> `show_statement_type ` is an abstract for str "show str force_eof", str can be ID,reserved_keyword and non_reserved_keyword. I think this is more clear for code.

`show tables|show columns` are `reserved_keyword`, here may be lead to ambiguity. Especially with the word `force_eof`.